scripts/install/install_macos.sh

```diff
@@ -592,11 +613,11 @@ display_results()
 {
     banner 'Information'
     increase_indent
-    info "Agent Home:         $(fg_cyan "$INSTALL_DIR")$(reset)"
-    info "Agent Config:       $(fg_cyan "$INSTALL_DIR/config.yaml")$(reset)"
-    info "Start Command:      $(fg_cyan "sudo launchctl load /Library/LaunchDaemons/$SERVICE_NAME.plist")$(reset)"
-    info "Stop Command:       $(fg_cyan "sudo launchctl unload /Library/LaunchDaemons/$SERVICE_NAME.plist")$(reset)"
-    info "Logs Command:       $(fg_cyan "sudo tail -F $INSTALL_DIR/log/collector.log")$(reset)"
+    info "Agent Home:                    $(fg_cyan "$INSTALL_DIR")$(reset)"
+    info "Agent Config:                  $(fg_cyan "$INSTALL_DIR/effective.yaml")$(reset)"
+    info "Agent Logs Command:            $(fg_cyan "sudo tail -F $INSTALL_DIR/agent.log")$(reset)"
```

Comment:
This has changed, probably will change again too 😅 
```suggestion
    info "Agent Config:                  $(fg_cyan "$INSTALL_DIR/supervisor_storage/effective.yaml")$(reset)"
    info "Agent Logs Command:            $(fg_cyan "sudo tail -F $INSTALL_DIR/supervisor_storage/agent.log")$(reset)"
```